shim/src/realm.js

```diff
@@ -89,7 +89,7 @@ function setDefaultBindings(realmRec) {
   const intrinsics = realmRec[Intrinsics];
   const descs = getStdLib(intrinsics);
   for (const name of Object.getOwnPropertyNames(descs)) {
-    if (name !== 'Infinity' || name !== 'NaN' || name !== 'undefined') {
+    if (!(name in { Infinity, NaN, undefined })) {
```

Comment:
the previous check was better; this will incorrectly pass for toString, for example.

Reply:
Good point! (My bad)

In any case, the previous code had a deMorgan confusion. It needed to use `&&` rather than `||`. That's what we should return to.

Reply:
oops, good catch